receiver/postgresqlreceiver/scraper_test.go

```diff
@@ -439,6 +439,51 @@ func TestScrapeQuerySample(t *testing.T) {
 	assert.NoError(t, errs)
 }
 
+func TestScrapeQuerySampleWithTraceparent(t *testing.T) {
```

Comment:
Is there any possible way to define plaintext strings as constants to reuse in this method? This method is pretty unreadable in its current form, having to try to track order of arguments and values with so many columns in the query.

Reply:
Done